autogluon/utils/tabular/ml/trainer/abstract_trainer.py

```diff
@@ -520,25 +533,27 @@ def train_multi_levels(self, X_train, y_train, X_val, y_val, models: List[Abstra
                 time_limit_core = time_limit_for_level
                 time_limit_aux = max(time_limit_for_level * 0.1, min(time_limit, 360))  # Allows aux to go over time_limit, but only by a small amount
             if level == 0:
-                self.stack_new_level(X=X_train, y=y_train, X_val=X_val, y_val=y_val, models=models, level=level, hyperparameter_tune=hyperparameter_tune, feature_prune=feature_prune, time_limit_core=time_limit_core, time_limit_aux=time_limit_aux)
+                models = self.get_models(hyperparameters, hyperparameter_tune=hyperparameter_tune, level=level)
+                base_model_names, _ = self.stack_new_level(X=X_train, y=y_train, X_val=X_val, y_val=y_val, models=models, level=level, base_model_names=base_model_names, hyperparameter_tune=hyperparameter_tune, feature_prune=feature_prune, time_limit_core=time_limit_core, time_limit_aux=time_limit_aux)
             else:
-                self.stack_new_level(X=X_train, y=y_train, X_val=X_val, y_val=y_val, level=level, time_limit_core=time_limit_core, time_limit_aux=time_limit_aux)
+                models = self.get_models(hyperparameters, level=level)
+                base_model_names, _ = self.stack_new_level(X=X_train, y=y_train, X_val=X_val, y_val=y_val, models=models, level=level, base_model_names=base_model_names, time_limit_core=time_limit_core, time_limit_aux=time_limit_aux)
 
         self._time_limit = None
         self.save()
 
-    def stack_new_level(self, X, y, X_val=None, y_val=None, level=0, models=None, hyperparameter_tune=False, feature_prune=False, time_limit_core=None, time_limit_aux=None):
-        core_models = self.stack_new_level_core(X=X, y=y, X_val=X_val, y_val=y_val, models=models, level=level, hyperparameter_tune=hyperparameter_tune, feature_prune=feature_prune, time_limit=time_limit_core)
+    def stack_new_level(self, X, y, models: List[AbstractModel], X_val=None, y_val=None, level=0, base_model_names=None, hyperparameter_tune=False, feature_prune=False, time_limit_core=None, time_limit_aux=None):
```

Comment:
documentation could highlight how multiple stackers could be employed

Reply:
added